pkg/reconciler/v1alpha1/taskrun/taskrun.go

```diff
@@ -103,10 +104,17 @@ func NewController(
 	// 	DeleteFunc: impl.Enqueue,
 	// })
 
-	c.tracker = tracker.New(impl.EnqueueKey, 30*time.Minute)
+	c.tracker = tracker.New(impl.EnqueueKey, 30*time.Second)
 	buildInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc:    c.tracker.OnChanged,
 		UpdateFunc: controller.PassNew(c.tracker.OnChanged),
 	})
+
+	resourceInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
```

Comment:
Is there a reason we watching the updates and add for resources?
If we do that, would the `reconcile` method be called when a resource is updated or added?  We should probably support looking resources if that is the case.

Reply:
I added the `resourceInformer` to be able to Get the `Resource` as part of adding the `AddInputResource` call. This func needs to Get the `PipelineResource` to be able to add the Git input source to the Build

Reply:
> I added the resourceInformer to be able to Get the Resource as part of adding the AddInputResource call. 

I'm a bit confused by this too - could you still have the Lister without adding an event handler to the resource informer?

> If we do that, would the reconcile method be called when a resource is updated or added? We should probably support looking resources if that is the case.

I think this would only be the case if Reconcile actually modified Resources, but I don't think it does? (might have missed it tho)

Reply:
I'll see what happens if I just use the lister, should be okay without this